lib/translations_manager/locales.rb

```diff
@@ -23,6 +23,8 @@ module TranslationsManager
     'fr',
     'gl',
     'he',
+    'hr',
+    'hr_HR',
```

Comment:
This list should only contain `hr`, not `hr_HR`.

Reply:
OK, changed that and amended the commit. Running `scripts/pull_translations.rb` in the main discourse repo threw an error when I removed `hr_HR`, but after deleting `config/locales/*hr_HR*` there, everything was fine.